packages/neo-one-typescript-concatenator/src/Concatenator.ts

```diff
@@ -87,11 +86,13 @@ export class Concatenator {
     }
 
     if (ts.isVariableStatement(node) && tsUtils.modifier.isNamedExport(node)) {
+      const symbol = this.context.getSymbol(node.declarationList.declarations[0].name);
+
       return tsUtils.setOriginal(
         ts.createVariableStatement(
-          node.modifiers === undefined
-            ? undefined
-            : node.modifiers.filter((modifier) => modifier.kind !== ts.SyntaxKind.ExportKeyword),
+          symbol === undefined || !this.context.isGlobalSymbol(symbol, node.getSourceFile().fileName)
```

Comment:
nit: this reads kind of funky, better to switch it around - `symbol !== undefined && this.context.isGlobalSymbol(symbol) ? node.modifiers : this.filterModifiers(node.modifiers)`. I.e. if this is a global symbol, don't filter.

Reply:
Also, this should be moved up to just after the `if` condition. If we aren't filtering the modifiers there's no reason to create a new variable statement.